src/main/java/org/folio/mr/service/impl/MediatedRequestActionsServiceImpl.java

```diff
@@ -121,4 +161,34 @@ private static MediatedRequestWorkflowLog buildMediatedRequestWorkflowLog(
     return log;
   }
 
+  private boolean localInstanceExists(MediatedRequestEntity mediatedRequest) {
```

Comment:
There exists many discussions regarding method order. I think it's always easier to read class whenever private methods are below their callers in the order they are called (or generally speaking they are placed in the call stack) but not at the end of file.